azure-proto-core/GenericResourcesOperations.cs

```diff
@@ -0,0 +1,49 @@
+using Azure;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace azure_proto_core
+{
+    public abstract class GenericResourcesOperations<TOperations, TResource> : OperationsBase 
```

Comment:
Why not just ResourcesOperations?  Also, I like ResourceOperations better, unless there is a naming rule that prevents this.